src/components/accordion/accordion.stories.mdx

```diff
@@ -541,5 +536,12 @@ Accordion styles can be easily overriden by providing `styleOverride` prop with
   </Story>
 </Preview>
 
+## Props
+
+### Accordion
+
 <Props of={Accordion} />
-<StyledSystemProps of={Accordion} spacing />
+
+### Style System 
```

Comment:
I think that instead of having :
<Props of={Accordion} />
<StyledSystemProps noHeader spacing />

We should just have:
<StyledSystemProps of={Accordion} spacing />